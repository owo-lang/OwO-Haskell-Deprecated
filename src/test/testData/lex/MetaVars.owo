_
_a
_123

__
___
_'
