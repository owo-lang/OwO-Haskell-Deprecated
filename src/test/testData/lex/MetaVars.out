MetaVar _ (0 1 1) (1 1 2)
MetaVar _a (2 2 1) (4 2 3)
MetaVar _123 (5 3 1) (9 3 5)
MetaVar __ (11 5 1) (13 5 3)
MetaVar ___ (14 6 1) (17 6 4)
MetaVar _' (18 7 1) (20 7 3)
